metadata.md

```diff
@@ -159,11 +165,15 @@ The site of wastewater sampling, including several *defaults* that can be used t
     -   `SAPFlowWater`: Sewer access point flowing water
```

Comment:
Issue 1: Does `siteType` have two different concepts that need separate variables?

What about:

 -  **siteType**: Type of site or institution where sample was taken.
      -   `airplane` : `WWTP`: Wastewater treatment plant
      -   `lagoon`: Lagoon
     -    `longTermCareHome`: 
     -   `sewageTruck` : 
      -   ....

- **accessType**: Where the sample was collected at the site.
    -   `SAPFlowWater`: Sewer access point flowing water
    -   `SAPStandingWater`: Sewer access point standing water
    -   `treatPlantInfluent`: Treatment plant influent
    -   `treatPlantPrimarySludge`: Primary treatment sludge
    -   `treatPlantEffluent`: Treatment plant effluent
    -   `buildingCleanout`: Building cleanout
    -   `propertyLineCleanout`: Property line cleanout

    -   ....

Not change to `measureType`.

-   **measureType**:  Statistical measures used to report the sample units of Ct/Cq, unless otherwise stated. Each measureType has a corresponding value (measureValue).

Reply:
Re siteType I am unsure, need to think about it, but you very well might be right.

re : measureType, I'm unsure what you mean.

Reply:
measureType. No change needed. Just pointing out there are fairly related measures.

Reply:
OK. Good video chat. I've pushed the changes.

Reply:
See https://github.com/Big-Life-Lab/covid-19-wastewater/pull/44/commits/d98bb5f9280ef9aeb10084e51feb21545273439b